src/main/kotlin/racingcar/InputView.kt

```diff
@@ -0,0 +1,24 @@
+package racingcar
+
+data class InputView(
+    val cars: List<Car> = readCars("자동차 대수는 몇 대인가요?"),
+    val roundCount: Int = readRoundCount("시도할 횟수는 몇 회인가요?"),
+) {
+    companion object {
+        private fun readCars(message: String): List<Car> {
+            println(message)
+
+            val list = mutableListOf<Car>()
+            repeat(readln().toInt()) {
+                list.add(Car())
+            }
+
+            return list
+        }
+
+        private fun readRoundCount(message: String): Int {
+            println(message)
+            return readln().toInt()
+        }
```

Comment:
두 함수 모두 메서드 파라미터로 메시지를 받을 필요가 없는 것 같아요. 🤔 

```suggestion
        private fun readCars(): List<Car> {
            println("자동차 대수는 몇 대인가요?")

            val list = mutableListOf<Car>()
            repeat(readln().toInt()) {
                list.add(Car())
            }

            return list
        }

        private fun readRoundCount(): Int {
            println("시도할 횟수는 몇 회인가요?")
            return readln().toInt()
        }
```